### ISSUE TITLE
Failed to compile Python app

### ISSUE THREAD

**Issue by ugideepak** [Mandal-21/Flight-Price-Prediction#10]
When i was trying to deploy this project on heroku its showing  "Push rejected, failed to compile python app. Push failed".
Please help me.

**Comment by Mandal-21** [comment]
This might be due to large files. Check the total size of you saved model. Heroku only provides 500MB of storage

**Comment by mohanhu** [comment]
same error to me

**Comment by pratik412** [comment]
No matching distribution found for pandas==1.0.1

**Comment by TilijaHari** [comment]
> No matching distribution found for pandas==1.0.1

> you can comment (#pandas==1.0.1) in requirements.txt and try to uninstall if you have any version of pandas and install the necessary ones.

**Comment by Siddharth540** [comment]
app.py giving error "no module found named flask-cors , even when i have intalled flask-cors in conda prompt

**Comment by TilijaHari** [comment]
> app.py giving error "no module found named flask-cors , even when i have intalled flask-cors in conda prompt

Please do check the environment. Have you installed the flask-cors in the same environment or not.
You can also do it through your Pycharm terminal as well in that case. To be honest you can solve this on your own. Believe in yourself and Google.

**Comment by Siddharth540** [comment]
Yes I have installed it in conda terminal but still it is not working and
also i am trying to deploy in heroku it  is also giving error saying "
encountered error while trying to install package numpy "
Please help
